tests/test_api.py

```diff
@@ -0,0 +1,129 @@
+def test_create_snippet(fastapi_client):
+    response = fastapi_client.post(
+        "/snippets/",
+        json={
+            "title": "first snippet",
+            "code": "print('hello world')",
+            "description": "first snippet in snipster",
+            "language": "python",
+            "tags": "python, fastapi",
+            "favorite": True,
+        },
+    )
+    assert response.status_code == 201
+    assert "first snippet" == response.json()["title"]
```

Comment:
I see you do 'actual assert expected' elsewhere which I find best, so here I would swap the too